.yarn/patches/vite-npm-5.0.0-beta.4-47b562f0aa.patch

```diff
@@ -0,0 +1,373 @@
+diff --git a/dist/node/chunks/dep-69dc786c.js b/dist/node/chunks/dep-69dc786c.js
+index 1c4ec28da1ac1d85a740ab5970b49a5774a3765d..b8f3034e833be8fa13357dd0bf53d3bb1fe223f8 100644
+--- a/dist/node/chunks/dep-69dc786c.js
```

Comment:
cant say i like this, it will break every vitest/vite update. is this new to the beta? why did you use the beta?

also most of the changes are whitespace, would be good to at least get rid of those so its clear what actually changed (which is basically one line)

Reply:
Honestly... I can just remove this patch bc its useless (reminds me I forgot to document one more quirk with vitest testing). Will do